scripts/dot-properties/generate.py

```diff
@@ -9,27 +9,45 @@
 LOGGER = logging.getLogger('generate')
 
 
-def _get_connection(profile_name, region_name):
-    LOGGER.info('Connecting to AWS using Profile: {profile} in Region {region}'.format(profile=profile_name,
-                                                                                       region=region_name))
-    return cloudformation.connect_to_region(region_name=region_name, profile_name=profile_name)
+def _boto_session(aws_profile_name):
+    LOGGER.info('Using AWS profile {}'.format(aws_profile_name))
+    boto3.setup_default_session(profile_name=aws_profile_name)
 
 
-def _get_stack_outputs(connection, stack_name_or_id):
-    stack = connection.describe_stacks(stack_name_or_id=stack_name_or_id)
+def _get_stack_name():
+    user = boto3.resource('iam').CurrentUser()
+    stack_name = 'media-service-DEV-{}'.format(user.user_name)
+    LOGGER.info('Using stack {}'.format(stack_name))
+    return stack_name
 
-    if stack:
-        LOGGER.info('Stack found: {stack}'.format(stack=stack_name_or_id))
-        outputs = {o.key: o.value for o in stack[0].outputs}
+
+def _get_client():
+    return boto3.client('cloudformation')
+
+
+def _get_stack_outputs():
+    stack_name = _get_stack_name()
+    cf_client = _get_client()
+    stack = cf_client.describe_stacks(StackName=stack_name)
+
+    outputs = {}
+
+    LOGGER.info('Here is your CloudFormation Stack output')
+
+    for output in stack['Stacks'][0]['Outputs']:
+        key = output['OutputKey']
+        value = output['OutputValue']
 
         # Cropper service doesn't expect a protocol to be included,
         # remove the http:// string from the ImageOriginWebsite item.
-        outputs['ImageOriginWebsite'] = outputs['ImageOriginWebsite'].replace('http://', '')
+        if key == 'ImageOriginWebsite':
+            value = value.replace('http://', '')
+
+        LOGGER.info('{key} = {value}'.format(key=key, value=value))
 
-        return outputs
-    else:
-        LOGGER.warn('Stack not found: {stack}'.format(stack=stack_name_or_id))
```

Comment:
boto3 will raise a nicely formed exception if the stack doesn't exist, so we don't need to log anything.